templates/coderd.yaml

```diff
@@ -128,7 +144,11 @@ spec:
             - name: STORAGE_CLASS
               value: {{ .Values.storageClassName | quote }}
             - name: TURN_ENABLED
+              {{- if hasKey .Values "cemanager" }}
               value: {{ .Values.cemanager.turn.enable | quote }}
+              {{- else }}
+              value: {{ .Values.coderd.turn.enable | quote }}
```

Comment:
With `coderd`, TURN port shouldn't be optional IMO. If they move to the new pattern that also enables TURN by default I think

Reply:
Good point!